recipe/meta.yaml

```diff
@@ -350,7 +350,7 @@ outputs:
         - imagemagick >=7.1.1_21
         - ipdb >=0.13.13
         - ipympl >=0.9.4
-        - ipython >=8.22.2
+        - ipython >=8.22.2,<9
```

Comment:
The Jira ticket says 9.0.0 is broken but is the implication that 9.0.1 might be okay and it was a packaging issue?

Reply:
Maybe?  I don't know, I haven't followed up too deeply.